File: app/src/main/java/com/example/ctap/fido2/AuthenticatorDisplay.java
package com.example.ctap.fido2;

import java.util.concurrent.CompletableFuture;

public interface AuthenticatorDisplay {
    CompletableFuture<Boolean> confirmMakeCredentials(
            final PublicKeyCredentialRpEntity rp,
            final PublicKeyCredentialUserEntity user
    );


}

File: app/src/main/java/com/example/ctap/fido2/AuthenticatorDisplay.kt
package com.example.ctap.fido2

import java.util.concurrent.CompletableFuture

interface AuthenticatorDisplay {
    fun confirmMakeCredentials(
            rp: PublicKeyCredentialRpEntity,
            user: PublicKeyCredentialUserEntity
    ): CompletableFuture<Boolean>
}
